### PR TITLE
Simple regression

### PR DESCRIPTION
**Pull request by stevekogo**
Implemented residual plots

### CODE REVIEW

**Comment by dannyparsons** [comment]
Hi Steve, this is looking really great! Well done getting the ggfortify working. I think the easiest way to now give you feedback on this is to ask @rdstern to now play around with the dialog and subdialog and he can comment on how it works and any improvements. I'll make sure he can get the latest code and then he'll get in touch with you.

**Comment by dannyparsons** [comment]
You probably won't hear from Roger until the afternoon. If you need other things to work on, Roger suggests some of the other stats stuff he has wrote to everyone about, e.g. principal components. But if you already have things to do then that's fine too.

**Comment by rdstern** [comment]
Hi Steve,

I am looking forward to trying the regression dialogue, once Danny comes in to work.

If you have lots of other work then fine.  If you are wondering what to do next, then here are two ideas.

The first is to continue with the regression dialogues.  The one with a factor should be easy to do, and I have described it in an e-mail.  Happy also to talk it over if that is a direction you would like to go in.  Then there is the main one – Fit Model.  For now, with that one, just do an ordinary text box for the model to be typed in.  Danny is working on the tests for a text box in general.

The second would be to try principal components.  That should be simple – in R – and the ggfortify also works for that!

Cheers

Roger

**Comment by stevekogo** [comment]
Dear Roger,

Hope you are doing well. John has taken the on PCA and I intend to work on
regression with facors once I am done with plotting fitted model on simple
regression.

I also intend to work on CCA in the course of the week.

Cheers,

Steve

File: instat/static/InstatObject/R/Rsetup.R [hunk 1]
```diff
@@ -7,3 +7,4 @@ library(ggplot2)
 library(extRemes)
 # TODO do we need reshape? It causes conflicts with reshape2 but this can be avoided if reshape is really needed
 #library(reshape)
+library(ggfortify)
```
Comment (dannyparsons) [inline instat/static/InstatObject/R/Rsetup.R]: @africanmathsinitiative/instat Everyone will need to add this package now. 
Remember to inform everyone when you add a new package in R.

Comment (stevekogo) [inline instat/static/InstatObject/R/Rsetup.R]: Dear all,

Sorry for not informing you about adding *ggfortify* package.
Ensure that you have it.

Regards,

Steve

**Pull request merged**